contracts/adapters/swap/dexter/src/contract.rs

```diff
@@ -128,25 +119,9 @@ pub fn execute(
     match msg {
         ExecuteMsg::Receive(cw20_msg) => receive_cw20(deps, env, info, cw20_msg),
         ExecuteMsg::Swap { routes } => {
-            if routes.len() != 1 {
-                return Err(ContractError::Skip(SkipError::MustBeSingleRoute));
-            }
-
-            let operations = routes.first().unwrap().operations.clone();
-
-            // validate that there's at least one swap operation
-            if operations.is_empty() {
-                return Err(ContractError::SwapOperationsEmpty);
-            }
-
-            let coin = one_coin(&info)?;
+            one_coin(&info)?;
 
-            // validate that the one coin is the same as the first swap operation's denom in
-            if coin.denom != operations.first().unwrap().denom_in {
```

Comment:
Do we do validation elsewhere that makes the validations you deleted no longer be relevant? I don't seem to see any validation but may be missing it

Reply:
Oh we do validation in entrypoint so we shouldn't need this... I see